x-pack/plugins/actions/server/builtin_action_types/email.ts

```diff
@@ -20,10 +19,10 @@ import { ActionsConfigurationUtilities } from '../actions_config';
 export type ActionTypeConfigType = TypeOf<typeof ConfigSchema>;
 
 const ConfigSchemaProps = {
-  service: nullableType(schema.string()),
```

Comment:
the `nullableType` function was a pre-cursor to having `schema.nullable()` available, so I removed it here.